src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,53 @@
+package duke.task;
+
+import duke.file.FileWritable;
+
+public abstract class Task implements FileWritable {
+    private static final String ICON_TICK = "\u2713";
+    private static final String ICON_CROSS = "\u2718";
+    private static final String STATUS_OPEN_BRACKET = "[";
+    private static final String STATUS_CLOSE_BRACKET = "] ";
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        isDone = false;
+    }
+
+    public Task(Boolean done, String description) {
+        this.description = description;
+        isDone = done;
+    }
+
+    /**
+     * Sets the task done boolean to the parameter value.
+     *
+     * @param done boolean status of the task.
+     */
+    public void setDone(boolean done) {
```

Comment:
`Boolean` variables should be named to sound like booleans(i.e. isDone, isSet etc.) Please consider changing the name of your boolean variable.

Reply:
Thanks for pointing it out to me!